### PR TITLE
Upgrade terraform-provider-confluent to v1.25.0

### LINKED ISSUE #54: Upgrading a cluster (Basic -> Standard) does not work (no diff)
### What happened?

Changing a cluster type does not produce a diff.

### Steps to reproduce

Declare a cluster:

```typescript
new confluent.KafkaCluster("my-cluster", {
  displayName: "inventory",
  availability: "SINGLE_ZONE",
  cloud,
  region,
  environment: {
    id: env.id,
  },
  basics: [{}]
});
```

Run `pulumi up`.

Change its type (the awkward syntax is noted in #47):

```
new confluent.KafkaCluster("my-cluster", {
  displayName: "inventory",
  availability: "SINGLE_ZONE",
  cloud,
  region,
  environment: {
    id: env.id,
  },
  standards: [{}] // This is what changed
});
```

Run `pulumi preview`.

### Expected Behavior

A diff should be noted.

### Actual Behavior

No diff appears.

### Output of `pulumi about`

_No response_

### Additional context

Version 1.13 of the TF provider does not exhibit this behavior.

### Contributing

Vote on this issue by adding a 👍 reaction. 
To contribute a fix for this issue, leave a comment (and link to your pull request, if you've opened one already).

### LINKED ISSUE #56: Upgrade terraform-provider-confluent to v1.20.0
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.20.0

### LINKED ISSUE #58: Upgrade terraform-provider-confluent to v1.21.0
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.21.0

### LINKED ISSUE #59: Upgrade terraform-provider-confluent to v1.21.1
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.21.1

### LINKED ISSUE #60: Upgrade terraform-provider-confluent to v1.22.0
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.22.0

### LINKED ISSUE #61: Upgrade terraform-provider-confluent to v1.22.1
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.22.1

### LINKED ISSUE #62: Upgrade terraform-provider-confluent to v1.23.0
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.23.0

### LINKED ISSUE #63: Upgrade terraform-provider-confluent to v1.23.1
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.23.1

### LINKED ISSUE #64: Upgrade terraform-provider-confluent to v1.24.0
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.24.0

### LINKED ISSUE #67: Upgrade terraform-provider-confluent to v1.25.0
Release details: https://github.com/confluentinc/terraform-provider-confluent/releases/tag/v1.25.0

### PR DESCRIPTION
**Pull request by aq17**
Fixes #67 
Fixes #64 
Fixes #63 
Fixes #62 
Fixes #61 
Fixes #60 
Fixes #59 
Fixes #58 
Fixes #56

### CODE REVIEW

File: provider/resources.go [hunk 1]
```diff
@@ -95,30 +95,47 @@ func Provider() tfbridge.ProviderInfo {
...
-			"confluent_schema_registry_cluster":    {Tok: tfbridge.MakeResource(mainPkg, mainMod, "SchemaRegistryCluster")},
-			"confluent_transit_gateway_attachment": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "TransitGatewayAttachment")},
+			"confluent_identity_pool":     {Tok: tfbridge.MakeResource(mainPkg, mainMod, "IdentityPool")},
+			"confluent_identity_provider": {Tok: tfbridge.MakeResource(mainPkg, mainMod, "IdentityProvider")},
+			"confluent_ksql_cluster":      {Tok: tfbridge.MakeResource(mainPkg, mainMod, "KsqlCluster")},
+			"confluent_schema": {
+				Tok: tfbridge.MakeResource(mainPkg, mainMod, "Schema"),
+				Fields: map[string]*tfbridge.SchemaInfo{
+					"schema": {
+						CSharpName: "SchemaDetails",
```
Comment (aq17) [inline provider/resources.go:105]: Not sure if adding `Details` is the convention for C# conflicts 
cc @jazzyfresh

**Comment by iwahbe** [comment]
@aq17 can you point to what part of this PR fixes https://github.com/pulumi/pulumi-confluentcloud/issues/54?

**Comment by aq17** [comment]
> @aq17 can you point to what part of this PR fixes #54?

My bad, that is a mistake

**Review by iwahbe — APPROVED** [review]
Looks good to me.

**Pull request merged**